packages/forms-web-app/src/pages/process-guide/_translations/cy.json

```diff
@@ -0,0 +1,40 @@
+{
+	"preApplication": {
+		"subHeading": "Cyn-ymgeisio",
```

Comment:
Can this and the additional stages be named "heading"? These translations will be used shared and used as the headings on each of the subsequent process guide pages?